database/join_questions_answes.sql

```diff
@@ -0,0 +1,21 @@
+USE `labby.db`;
+DROP procedure IF EXISTS `join_questions_answers`;
+
+DELIMITER $$
+ 
+CREATE PROCEDURE `join_questions_answers` ()
```

Comment:
Awesome thanks! Also one small thing, can we name this something and the `drop procedure` above something else so the function call name is more descriptive? Like: 

```suggestion
CREATE PROCEDURE `load_questions_answers` ()
```